src/App.tsx

```diff
@@ -1,8 +1,17 @@
-import React from 'react';
+import React, { useState } from 'react';
+import cn from 'classnames';
 import 'bulma/css/bulma.css';
 import './App.scss';
 
-export const goodsFromServer = [
+type Good = string;
+
+enum Sort {
+  DEFAULT = '',
+  ALPHABETICALLY = 'sort alphabetically',
+  BY_LENGTH = 'sort by length',
```

Comment:
Maybe better to use CamelCase here?

Reply:
@kaavka Such a view of the record as now is also possible.